lib/loadEnvs.js

```diff
@@ -3,21 +3,14 @@ const fieldsEnvMap = {
   'baseURL': 'BASE_URL',
   'clientID': 'CLIENT_ID',
   'clientSecret': 'CLIENT_SECRET',
+  'sessionSecret': 'SESSION_SECRET',
```

Comment:
`SESSION_SECRET` is the env name for the secret that encrypts the cookie. If this env exists then the config value does not need to be passed to the SDK config.